VTUL/VIVO_VT_Theme
# prevent Open Access links from wrapping

**Prevents Open Access links from wrapping across lines.**
* * *

**JIRA Ticket**: (https://webapps.es.vt.edu/jira/browse/LIBTD-1688) (:star:)

# What does this Pull Request do? (:star:)
Prevents Open Access links from wrapping across lines.

# What's the changes? (:star:)
Adds `white-space: nowrap;` to the .handleid class to prevent links from wrapping across lines.

Without `nowrap`:
<img width="851" alt="screen shot 2019-01-29 at 11 59 27 am" src="https://user-images.githubusercontent.com/1202435/51925573-5a29f100-23bd-11e9-8341-327c4576f1b2.png">

With `nowrap`:
<img width="851" alt="screen shot 2019-01-29 at 12 00 04 pm" src="https://user-images.githubusercontent.com/1202435/51925604-66ae4980-23bd-11e9-926a-3aa69733e3ab.png">


# How should this be tested?
* Create or import articles of various lengths similar to the ones on this page: https://vivo-dev-theme.cloud.lib.vt.edu/display/vpannabe
* Ensure that none of the Open Access links are split across lines.

# Additional Notes:
* Branch: `LIBTD-1688`

# Interested parties
@yinlinchen 

(:star:) Required fields
